Misc/NEWS.d/next/Core_and_Builtins/2024-09-25-13-45-01.gh-issue-124445.zfsD7q.rst

```diff
@@ -0,0 +1 @@
+Allow specializing Generic ParamSpec aliases
```

Comment:
This isn't very clear. Something like "Fix specialization of generic aliases that are generic over a :class:`typing.ParamSpec` and have been specialized with a nested type variable."

Reply:
Thanks! That's much clearer :slightly_smiling_face: